Frontend/ui-driver/src/Strings/BN.js

```diff
@@ -767,5 +767,12 @@ const bengaliStrings = {
 	NEED_HELP_JOINING_THE_PLAN : "পরিকল্পনা বা অটোপেতে যোগদানের জন্য সহায়তা দরকার?",
   	NEED_HELP_CALL_SUPPORT : "সাহায্য দরকার? <span style='color:#2194FF'>কল সমর্থন</span>",
 	SETUP_AUTOPAY_NOW_TO_GET_SPECIAL_DISCOUNTS  : "বিশেষ ছাড় পেতে এখনই অটোপে সেটআপ করুন",
-	SETUP_NOW : "এখন সেটআপ"
-	}
+	SETUP_NOW : "এখন সেটআপ",
+	NEED_HELP_JOINING_THE_PLAN : "পরিকল্পনায় যোগদানের জন্য সাহায্যের প্রয়োজন? যোগাযোগ করুন",
```

Comment:
why is format messed up, few strings are replicated again. Like - NEED_HELP_CALL_SUPPORT can see multiple occurrences of the same string

Reply:
fixed that but alignments are showing correctly in VS code but not in git. I don't know why